src/test/codegen/branch-protection.rs

```diff
@@ -1,12 +1,12 @@
 // Test that the correct module flags are emitted with different branch protection flags.
 
-// revisions: bti pac-ret leaf b-key
+// revisions: BTI PACRET LEAF BKEY NONE
 // min-llvm-version: 12.0.0
 // needs-llvm-components: aarch64
-// [bti] compile-flags: -Z branch-protection=bti
-// [pac-ret] compile-flags: -Z branch-protection=pac-ret
-// [leaf] compile-flags: -Z branch-protection=pac-ret,leaf
-// [b-key] compile-flags: -Z branch-protection=pac-ret,b-key
+// [BTI] compile-flags: -Z branch-protection=bti
```

Comment:
Is it worth also testing the empty "`-Z branch-protection=`"? If I understand correctly, that should result in module flags being generated (with zero values), and that behaviour will be useful for validation before stabilisation.

Reply:
Turns out no, empty value is not valid. Implementing that would be out of scope for this PR I feel.